spec/factories/client.rb

```diff
@@ -18,6 +18,29 @@
     role_name { "ROLE_DATACENTRE" }
     password_input { "12345" }
     is_active { true }
+    subjects do
+      [
+        {
+          classificationCode: "1001",
+          schemeUri: "http://example.com/schemeUri",
+          subject: "Example Subject",
+          subjectScheme: "Example Subject Scheme (ESS)",
```

Comment:
Given that this is explicitly _not_ FOS, is it worth having slightly different values for the code and subject text so that they can be asserted against individually in tests without needing to also assert against the subjectScheme field?